stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -77,50 +77,19 @@ public struct ${Self}<Pointee>
     self._rawValue = Builtin.inttoptr_Word(bitPattern._builtinWordValue)
   }
 
-  /// Convert from any `UnsafeMutablePointer`, possibly with a
```

Comment:
Ah, yes I forgot to talk about this in the commit message.

I originally marked these as unavailable, then found that it was actually interfering with more useful diagnostics w.r.t overload resolution. Let me experiment with that more in a different PR. Either way, I'll see about adding migrator support for important cases.